trio/tests/test_exports.py

```diff
@@ -49,6 +49,11 @@ def public_namespaces(module):
 NAMESPACES = list(public_namespaces(trio))
 
 
+# It doesn't make sense for downstreams to run this test, since they might be
```

Comment:
s/downstreams/packagers/ ?

Reply:
Following the terminology @njsmith suggested, I changed this to "upstream distributors". Is that clearer? I also adjusted some of the other wording in this paragraph, and likewise in the commit message.